client/assets/components/navbar.scss

```diff
@@ -0,0 +1,171 @@
+.top-navbar {
+  height: 70px;
+  border-radius: 0;
+  background-color: #fff;
+  margin: 0;
+  border: 0;
+  .navbar-brand {
+    height: 70px;
+    width: 70px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+}
+.navbar-collapse.collapse {
+  display: flex !important;
+  height: 70px !important;
+}
+.top-navbar__right-container {
+  align-items: center;
+  justify-content: right;
+  padding: 0;
+  .nav-profile {
+    width: 183.5px;
+    &__list-menus {
+      display: flex;
```

Comment:
### Critical

I'm seeing a lot of flex definition in a number of places, what do you think about having a mixin for `column-flex` and `row-flex` or an `@extend` function instead?